### PR TITLE
apps/plans: add pagination to plan dashboard list

### PR DESCRIPTION
**Pull request by sabinammm**

### CODE REVIEW

**Review by fuzzylogic2000 — COMMENTED** [review]
Yes, it works and that's how you add a pagination. 
And here's the but: we have a problem with group members. You can already try to recreate the problem with this info:
https://github.com/liqd/a4-meinberlin/issues/2616
https://github.com/liqd/a4-meinberlin/commit/8b573953d09efffb354f71aecec3ce9c6ca72185
And then let's talk after standup!

File: meinberlin/apps/plans/views.py [hunk 1]
```diff
@@ -104,7 +105,7 @@ def get_context_data(self, **kwargs):
 
         if (hasattr(settings, 'A4_USE_VECTORMAP') and
                 settings.A4_USE_VECTORMAP):
-            use_vector_map = 1
+            use_vector_map = 12
```
Comment (fuzzylogic2000) [inline meinberlin/apps/plans/views.py:108]: I am not sure, I was already allowed to look. But anyway: this shouldn't be changed. :)

**Review by fuzzylogic2000 — COMMENTED** [review]
Works like a charm! But the linter complains about the sorting of the imports.

**Comment by sabinammm** [comment]
> Works like a charm! But the linter complains about the sorting of the imports.

Yes, I've tried moving the import that I added to various locations...I do not understand.

File: meinberlin/apps/plans/views.py (hunk 1 above)
Comment (sabinammm) [reply meinberlin/apps/plans/views.py:108]: That was a mistake - I kept changing the pagination to check if it worked from 12 to 1. Sorry about this

**Comment by fuzzylogic2000** [comment]
> Yes, I've tried moving the import that I added to various locations...I do not understand.

It usually tells you where to put it:
```
 from adhocracy4.filters.filters import FreeTextFilter
 from adhocracy4.rules import mixins as rules_mixins
 from meinberlin.apps.contrib.views import CanonicalURLDetailView
+from meinberlin.apps.dashboard.mixins import DashboardProjectListGroupMixin
 from meinberlin.apps.maps.models import MapPreset
 from meinberlin.apps.organisations.models import Organisation
 from meinberlin.apps.plans.forms import PlanForm
 from meinberlin.apps.plans.models import Plan
-from meinberlin.apps.dashboard.mixins import DashboardProjectListGroupMixin
```

**Review by fuzzylogic2000 — COMMENTED** [review]
The import not working in the right place for you is so weird, I will try it out and change what I think should be changed. I will put it in an extra commit so that you can see the changes and then we squash again afterwards.

**Comment by fuzzylogic2000** [comment]
@sabinammm Let me know when you looked at it, then I'll update the commits and merge!

**Review by sabinammm — COMMENTED** [review]
Yes perfect! I'm just confused because I am so certain that I fixed my use_vector_map mixup and force pushed this.... the linter is not my friend today.

**Comment by fuzzylogic2000** [comment]
> Yes perfect! I'm just confused because I am so certain that I fixed my use_vector_map mixup and force pushed this.... the linter is not my friend today.

Yes, sometimes it just isn't. :woman_shrugging:

**Review by fuzzylogic2000 — APPROVED** [review]
Yeah!

**Pull request merged**